contracts/proxy/Clones.sol

```diff
@@ -17,6 +17,16 @@ pragma solidity ^0.8.19;
  * _Available since v3.4._
  */
 library Clones {
+    /**
+     * @dev A `CREATE` clone instance deployment failed.
+     */
+    error ERC1167FailedCreateClone();
```

Comment:
Thoughts on using "unsuccessful" instead of "failed"?

Reply:
`Unsuccessful` sounds less panicking than `Failed`, I think I'd go for Unsuccessful as long as we keep consistency. The current state is:

Errors with `Unsuccessful`:

- `SafeERC20UnsuccessfulOperation`

Errors with `Failed`:

- `GovernorFailedLowLevelCall`
- `TimelockFailedOperation`
- `ERC1167FailedCreateClone`
- `SafeERC20FailedLowLevelCall`
- `AddressFailedCall`
- `AddressFailedLowLevelCall`
- `Create2FailedDeployment`

What do you think?

Reply:
Are you suggesting all the errors above should use the same error word? I'm leaning towards Failed just because it's shorter.

Reply:
Yes.

My only preference towards Unsuccesful was because is more friendly.

I'll go with Failed as you suggested.